modules/infra/control/loopback.c

```diff
@@ -258,24 +237,12 @@ static void loopback_module_fini(struct event_base *) {
 	gr_pktmbuf_pool_release(loopback_pool, RTE_GRAPH_BURST_SIZE);
 }
 
-static void iface_loopback_to_api(void * /* info */, const struct iface * /* iface */) { }
-
-static struct iface_type iface_type_loopback = {
-	.id = GR_IFACE_TYPE_LOOPBACK,
-	.name = "loopback",
-	.info_size = sizeof(struct iface_info_loopback),
-	.init = iface_loopback_init,
-	.fini = iface_loopback_fini,
-	.to_api = iface_loopback_to_api,
-};
-
```

Comment:
If you rename everything s/loopback/vrf/, then this should stay in the same file.